protected/humhub/modules/admin/models/forms/FileSettingsForm.php

```diff
@@ -14,7 +14,7 @@ class FileSettingsForm extends \yii\base\Model
     public $maxFileSize;
     public $maxPreviewImageWidth;
     public $maxPreviewImageHeight;
-    public $hideImageFileInfo;
+    public $excludeMediaFilesPreview;
```

Comment:
The naming is not fitting since we show the preview of those files, but we do not include those files in the attachment file list. The preview is the actual image gallery/audio- videoplayer. How about `$excludeMediaFilesFromAttachments`?

Reply:
Actually, I choose `preview` because according to the codes, the `ShowFile` widget is responsible to render media files while `FilePreview` is responsible to list files in attachment and in codes, and their documents used the word `preview` for this usage.

Reply:
Makes sense, seems to be a general naming missmatch. Then maybe leave it as is, but perhaps add a phpdoc to clarify that attachment file list is meant.